src/Swarm/Language/Capability.hs

```diff
@@ -224,6 +224,7 @@ constCaps = \case
   Meet -> Just CMeet
   MeetAll -> Just CMeet
   Drill -> Just CDrill
+  Use -> Nothing -- Recipes alone shall dictate whether things can be "used"
```

Comment:
I will note that one possibly strange thing about this is that if you write a program like `build {move; use "flerb" forward; move}` it will fail, because the requirements analysis will not figure out that we should automatically equip a `flerb` when `build`ing the child robot.  You would have to explicitly write e.g. `build {move; require "flerb"; use "flerb" forward; move}`.  We might consider adding something to the requirements analysis which will at least work when `use` is applied to a statically known device name, as in this example.

Reply:
Created #1301